output.json

```diff
@@ -3999,6 +4037,308 @@
+            "description": "The unique identifier of the Workflow Run to which the Tasks belong.",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "An array of subset Task objects that were already started or completed, ordered by the created_at field, in ascending order.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "tasks": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "id": {
+                            "type": "string",
+                            "description": "The identifier for the Task."
+                          },
+                          "task_def_id": {
+                            "type": "string",
+                            "description": "The identifier for the Task Definition."
+                          },
+                          "created_at": {
+                            "type": "string",
+                            "format": "date-time",
+                            "description": "The date and time when the Task was created."
+                          },
+                          "updated_at": {
+                            "type": "string",
+                            "format": "date-time",
+                            "description": "The date and time when the Task was last updated."
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "default": {
+            "$ref": "#/paths/~1ping/get/responses/default"
+          }
+        }
+      }
+    },
+    "/workflow_runs/{workflow_run_id}/tasks/{task_id}": {
+      "get": {
+        "summary": "A single Task can be retrieved by calling this endpoint with the unique identifier of the Task and Workflow Run.",
+        "operationId": "retrieve_task",
+        "parameters": [
+          {
+            "name": "workflow_run_id",
+            "in": "path",
+            "required": true,
+            "description": "The unique identifier of the Workflow Run to which the Task belongs.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "task_id",
+            "in": "path",
+            "required": true,
+            "description": "The identifier of the Task you want to retrieve.",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A Task object.",
+            "content": {
+              "application/json": {
+                "schema": {
```

Comment:
In the retrieve task endpoint we return the full task object, so you can reference the Task object here.

The only place where we do not return the full task object is on the "list", which you have already fixed.